src/main/pg/pg_ids.h

```diff
@@ -152,7 +152,8 @@
 #define PG_MODE_ACTIVATION_CONFIG 553
 #define PG_DYN_NOTCH_CONFIG 554
 #define PG_RX_EXPRESSLRS_SPI_CONFIG 555
-#define PG_BETAFLIGHT_END 555
+#define PG_SCHEDULER_CONFIG 536
```

Comment:
```suggestion
#define PG_SCHEDULER_CONFIG 556
```

Reply:
Done, but also added whitespace to the other definitions to aid clarity.